### PR TITLE
docs(carousel): add demo for activeSlideChange

### LINKED ISSUE #3677: docs(carousel): add demo for activeSlideChange
related to #3232

### PR DESCRIPTION
**Pull request by artolshansky**
Add activeSlideChange listener for carousel (log in console after each switching slide)

close #3677

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Please, add more detailed description, how can I test this pr.

**Comment by YevheniiaMazur** [comment]
Please, turn off slides' autochanging and make your demo more usefull (add output instead of console log)
Now, due to your solution too much logs appears:
toomuch

File: demo/src/app/components/+carousel/demos/slide-changed-event/slide-changed-event.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-carousel-slide-changed-event',
+  templateUrl: './slide-changed-event.html'
+})
+export class DemoCarouselSlideChangedEventComponent {
+  slideModel: string = '';
```
Comment (IlyaSurmay) [inline demo/src/app/components/+carousel/demos/slide-changed-event/slide-changed-event.ts]: activeSlideChangeMessage or slideChangeMessage

Comment (IlyaSurmay) [reply demo/src/app/components/+carousel/demos/slide-changed-event/slide-changed-event.ts]: also, remove type annotation if a field has a default value

File: demo/src/app/components/+carousel/demos/slide-changed-event/slide-changed-event.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  slides: any[] = [
```
Comment (IlyaSurmay) [inline demo/src/app/components/+carousel/demos/slide-changed-event/slide-changed-event.ts]: same

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**